trustymail/cli.py

```diff
@@ -44,16 +57,39 @@ def main():
     else:
         timeout = 5
 
+    if args["--smtp-timeout"] is not None:
+        smtp_timeout = int(args["--smtp-timeout"])
+    else:
+        smtp_timeout = 5
+
+    if args["--smtp-localhost"] is not None:
+        smtp_localhost = args["--smtp-localhost"]
+    else:
+        smtp_localhost = None
+
+    if args["--smtp-ports"] is not None:
+        smtp_ports = list(map(lambda x:int(x), args["--smtp-ports"].split(',')))
```

Comment:
Forgive my new-ness to lambda's... Is this taking the arguments to the `--smtp-ports` options and returning a list of those numbers as `int`s? If so, perhaps a list comprehension is clearer?

```python
smtp_ports = [int(port) for port in args['--smtp-ports'].split(',')]
```

Reply:
Yes, that's all it's doing.  I will change it to a list comprehension.